build/Dockerfile

```diff
@@ -9,4 +9,4 @@ COPY requirements.yml ${HOME}/requirements.yml
 RUN ansible-galaxy collection install -r ${HOME}/requirements.yml \
  && chmod -R ug+rwx ${HOME}/.ansible
 
-RUN cp /etc/ansible/ansible.cfg ${HOME}/ansible-profiler.cfg && echo "callback_whitelist = profile_tasks" >> ${HOME}/ansible-profiler.cfg
+RUN cp /etc/ansible/ansible.cfg ${HOME}/ansible-profiler.cfg && echo "callback_enabled = profile_tasks" >> ${HOME}/ansible-profiler.cfg && echo "callback_whitelist = profile_tasks" >> ${HOME}/ansible-profiler.cfg
```

Comment:
I just want to comment on this. I left the callback_whitelist in here because the current base image of the ansible operator in use by Kiali today does not support the new callback_enabled setting yet. I put the callback_enabled in here now so it will be there when we do upgrade (having both won't hurt anything). It makes sense to do this as part of this PR. We can remove the callback_whitelist entry later when we no longer need it (i.e. when we are using an ansible base image that supports it).